pkg/controller/planexecution/planexecution_controller.go

```diff
@@ -305,10 +316,12 @@ func (r *ReconcilePlanExecution) Reconcile(request reconcile.Request) (reconcile
 		if !ok {
 			// Not specified in params
 			if param.Required {
+				planExecution.Status.State = kudov1alpha1.PhaseStateError
```

Comment:
If you're changing the `Status` here, you need to update the object with the client, it won't automatically get applied into the cluster

Reply:
```go
err = r.Update(context.TODO(), planExecution)
```

Reply:
:+1: